.github/workflows/nightly-rosetta-test.yaml

```diff
@@ -0,0 +1,45 @@
+name: Nightly Rosetta test
+
+on:
+  workflow_run:
+    workflows: [Nightly Rosetta build]
+    types: [completed]
+  workflow_dispatch:
+    inputs:
+      ROSETTA_IMAGE_TAG:
```

Comment:
Same as above, we will migrate to use complete image URLs instead of only the tag part.

Reply:
done